tfjs-backend-wasm/src/cc/kernels/Atan2.cc

```diff
@@ -0,0 +1,63 @@
+/* Copyright 2023 Google LLC. All Rights Reserved.
```

Comment:
we no longer add 'All Rights Reserved' to new files since it may conflict with our OSS license.
as https://github.com/tensorflow/tfjs/pull/7321#discussion_r1089585165